lib/gooddata/lcm/lcm2.rb

```diff
@@ -420,7 +403,7 @@ def check_unused_params(actions, params)
           end
 
           table = Terminal::Table.new :headings => ['Action', 'Parameter', 'Description', 'Parameter Type'], :rows => rows
-          puts table.to_s
+          GoodData.logger.info("\n#{table}")
```

Comment:
didn't you miss ".to_s" ? the logger calls inspect not to_s

Reply:
Rubocop was complaining because `#{table}` and `#{table.to_s}` is the same in string interpolation.

Reply:
Output is the same

test code:
```ruby
logger = Logger.new(STDOUT)
table = Terminal::Table.new :headings => ['Action', 'Parameter', 'Description', 'Parameter Type'], :rows => [['s','s','s','s']]
logger.info("\n#{table}")
```

output without `to_s`:
```
I, [2018-11-30T13:46:41.226898 #26665]  INFO -- : 
+--------+-----------+-------------+----------------+
| Action | Parameter | Description | Parameter Type |
+--------+-----------+-------------+----------------+
| s      | s         | s           | s              |
+--------+-----------+-------------+----------------+
```
output with `to_s`:
```
I, [2018-11-30T13:48:41.096952 #27500]  INFO -- : 
+--------+-----------+-------------+----------------+
| Action | Parameter | Description | Parameter Type |
+--------+-----------+-------------+----------------+
| s      | s         | s           | s              |
+--------+-----------+-------------+----------------+
```